SAGESharp.Tests/Animations/TCBQuaternionDataTests.cs

```diff
@@ -15,7 +15,7 @@ class TCBQuaternionDataTests
         [TestCaseSource(nameof(ComponentTestCases))]
         public void Test_Getting_The_Value_Of_A_Component(ComponentTestCase testCase)
         {
-            TCBQuaternionData quaternion = new TCBQuaternionData();
+            QuaternionKeyframe quaternion = new QuaternionKeyframe();
 
             testCase.SetPrivateField(quaternion, 0x8000);
```

Comment:
This test is failing because `0x8000` is implicitly casted to `int16`, cast it to `uint16`

Reply:
I changed the quaternion members to be signed instead of unsigned, as this is the correct behavior. The value should be signed. What do I do about that?